accessibility-scan.yml

```diff
@@ -0,0 +1,25 @@
+jobs:
+  - name: accessibility-scan
+    plan:
+    - task: run-pa11y
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: node
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            npm install -g pa11y-crawl
```

Comment:
should these commands be idempotent? i.e  not installing npm, pyenv, or jq if they are already in the system. It should check if the versions are up-to-date. Will do some investigation on how to do this.